securedrop/tests/functional/journalist_navigation_steps.py

```diff
@@ -628,18 +630,32 @@ def _visit_edit_account(self):
             'link-edit-account')
         edit_account_link.click()
 
-    def _visit_edit_secret(self, type):
+    def _visit_edit_secret(self, type, tooltip_text=''):
         reset_form = self.driver.find_elements_by_css_selector(
             '#reset-two-factor-' + type)[0]
         assert ('/account/reset-2fa-' + type in
                 reset_form.get_attribute('action'))
 
         reset_button = self.driver.find_elements_by_css_selector(
             '#button-reset-two-factor-' + type)[0]
+
+        # 2FA reset buttons show a tooltip with explanatory text on hover.
+        # Also, confirm the text on the tooltip is the correct one.
+        ActionChains(self.driver).move_to_element(reset_button).perform()
+        time.sleep(1)
+        explanatory_tooltip_opacity = self.driver.find_elements_by_css_selector(
+            '#button-reset-two-factor-' + type + ' span')[0].value_of_css_property('opacity')
+        explanatory_tooltip_content = self.driver.find_elements_by_css_selector(
+            '#button-reset-two-factor-' + type + ' span')[0].text
+
+        assert explanatory_tooltip_opacity == '1'
+        assert explanatory_tooltip_content == tooltip_text
```

Comment:
This assert will fail if the tests are run against a language other than English. Currently tests that assert the value of a string are only run if the default language is English, and skipped otherwise, like so:
```
if not hasattr(self, "accept_languages"):                           
                assert explanatory_tooltip_content == tooltip_text
````
This will allow the page-layout tests to run successfully, while also allowing the functional tests to run against different language settings.

(This probably means we have less functional test coverage for other locales, so it doesn't seem ideal, but that's a problem for another PR.)

Reply:
Makes sense. I will put this in place for now and maybe can check for the locale skipping part in a different PR.